ci/linkcheck.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then
+  FLAGS=""
+
+  echo "Doing full link check."
+else
+  CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r master..HEAD | tr '\n' ' ')
```

Comment:
Hmm, I think you want `master...HEAD` (three dots, not two) so that it shows a diff from the point where the branches diverged. What you have right now will diff from the tip of master.

```suggestion
  CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r master...HEAD | tr '\n' ' ')
```